### DISCUSSION TITLE
Auto-suggests of publisher and publication year when registering DOIs in DataCite Fabrica

### DISCUSSION THREAD

**Discussion by paulvierkant** [datacite/datacite-suggestions#124] [General Suggestions]
### What is the problem that your suggestion solves?

I manually have to choose the publisher and publication year.

### What solution might meet your needs?

Based on the majority of values of the other works in the same repository, suggest a value for publisher and publication year (also suggest the current year). I'd assume that the publisher is often the same in one repository. In case the suggestion is wrong, make the suggestion editable.
Same goes for Resource Type General and resource type. The form should suggest values based on the majority or last registration.

### Your name

Paul Vierkant

### Your organization

DataCite

### What group(s) would benefit from your suggestion?

- [X] DataCite members registering DOIs and metadata and administering accounts
- [ ] Developers building integrations with DataCite services
- [ ] Harvesters who index DataCite DOIs and metadata
- [ ] Individuals or institutions analyzing DataCite DOIs and metadata
- [X] DataCite staff
- [ ] Other

**Comment by paulvierkant** [comment]
Actually the same applies to the information in the following properties if they are in the same repository:
1. language
2. rights
3. format

**Comment by Angeliki367** [comment]
I agree with the suggestion. An auto-fill of values that are used very often will facilitate the whole process, especially when the list of publishers from the drop-down menu is so vast, resulting in real difficulties in picking and submitting the preferred option. I would say that autofill should be incorporated in other fields also, such as the data creators affiliations -if, of course, it is technically feasible.

**Reply by codycooperross** [reply]
Thank you, @Angeliki367 ! We hear you on the challenges of selecting among many entities in the publisher and affiliation drop-down menus, and we're looking toward ways to make this clearer and easier. The auto-suggest and auto-fill suggestion is also similar to a few other suggestions for enhancing the form and populating metadata in bulk:

- https://github.com/datacite/datacite-suggestions/discussions/107
- https://github.com/datacite/datacite-suggestions/discussions/109
